section4/reflection.md

```diff
@@ -1,21 +1,32 @@
   * List of states and their capitals
+  - This would be stored in an object as the capitals are characteristics that define a state and you can add more information to further identify a state.
   * List of things to pack for vacation
+  - This would be better to store this list in an array as it's a simple packing list storing single values for a packing list.
   * Names of all the Instagram accounts I follow
+  - this would be an array as it is a list of names who you follow on instagram and you can access their index in terms of who was your earliest / latest followers.
   * List of student names and their cohort
+  - This would be an object, because
   * Ingredients and amount of each ingredient to bake a cake
+  - This would be stored in an object because there are ingredients and measurements, which is similar to key-value pairs to store the information in how to bake a cake.
   * All my favorite restaurants
+  - I think this could go either way. An array would work if you just wanted to provide a list of your favorite restaurants. An object would come into play if you'd like to provide more details or characteristics to describe your favorite restaurants, such as location, cuisine, price, etc.
 
-1. In this section, we talked about an `transaction item object`. It has keys that represent different properties an bank transaction would have, and values that store the data. In a banking app, this data would eventually be displayed in the browser for the user. Think of another Object in a web application that you use frequently. What is it? What keys might be on it? What are example values? 
+1. In this section, we talked about an `transaction item object`. It has keys that represent different properties an bank transaction would have, and values that store the data. In a banking app, this data would eventually be displayed in the browser for the user. Think of another Object in a web application that you use frequently. What is it? What keys might be on it? What are example values?
+- Amazon stores a ton of data when it comes to purchases made on the app with similar keys as the transaction item object. For example, it could have keys like price, orderDate, primeCustomer, confirmation, etc. Their ability to track their customer's use and purchases through Amazon helps cater their product more for the user. 
 
 1. What questions do you still have about classes and/or Objects?
+- What's the difference between methods as a function that are stored inside a key value pair and behavior methods?
+- is there a resource for the functions / behavior methods that we can use to to understand what we can use in objects to manipulate the data? I had a hard time completing the last exercises (classMethods,js and tweet.js) because of the use of array methods as well as some syntax from loop iterations.
```

Comment:
Great reflections and questions from section4 (and the other sections too). Don't stress this too much—you'll get a lot more practice and instruction on these topics in Mod 1. 

If you want to dig in more during the downtime before Mod 1 starts, definitely look to the Mod 0 extensions and reach out to Amy if you could use more specific recommendations :)